seatunnel-connectors-v2/connector-pulsar/src/main/java/org/apache/seatunnel/connectors/seatunnel/pulsar/source/enumerator/PulsarSplitEnumerator.java

```diff
@@ -127,6 +129,19 @@ public void open() {
 
     @Override
     public void run() throws Exception {
+        if (partitionDiscoveryIntervalMs > 0) {
+            new ScheduledThreadPoolExecutor(1, runnable -> {
```

Comment:
How to close this thread pool when terminates a task?

Reply:
Sorry, I forgot it.